fili-core/src/main/java/com/yahoo/bard/webservice/application/AbstractBinderFactory.java

```diff
@@ -268,6 +268,8 @@ protected void configure() {
 
                 DataSourceMetadataService dataSourceMetadataService = buildDataSourceMetadataService();
 
+                bind(dataSourceMetadataService).to(DataSourceMetadataService.class);
```

Comment:
We constructed it and passed it to objects which we bound separately or loaded directly into tasks
line 275: queryBindingService
line 290, datasource metadata loader